ws-client/src/client.rs

```diff
@@ -80,15 +81,17 @@ impl ErrorFromBack {
 /// It's possible that the background thread is terminated and this makes the client unusable.
 /// An error [`Error::RestartNeeded`] is returned if this happens and users has to manually
 /// handle dropping and restarting a new client.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
we don't want this to be clone anymore for two reasons:

1. Uniform API with the `HTTP client`
2. Indicate to users that `Clone` means sharing resources such that users has to put in a proper such as in `Arc` or `Mutex` or whatever fits the particular usecase.